resources/javascripts/login.js

```diff
@@ -0,0 +1,56 @@
+var validation = false;
+
+callback = function (flag, id, value) {
+    validation = flag;
+    document.getElementById(id).innerHTML = value;
+}
+
+
+document.getElementById("submit-login").addEventListener("click", function () {

```

Comment:
for clarity, have the definition of any callback used in the application in a separate named function